scripts/fortran_tools/offline_check_fortran_vs_metadata.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+
+"""
+Recursively compare all fortran and metadata files in user-supplied directory, and report any problems
+USAGE: set PYTHONPATH environment variable to the the scripts directory, then:
+    python offline_check_fortran_vs_metadata.py --directory <full path to directory with scheme files> (--debug)
+"""
+
+
+import sys
+import os
+import logging
+import argparse
+
+# CCPP framework imports
```

Comment:
Could you try to add the scripts directory to keep from having to manipulate `PYTHONPATH`?
```
import site
site.addsitedir(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Reply:
good call -added!